package.json

```diff
@@ -89,6 +90,7 @@
     "jquery.dotdotdot": "github:amywieliczka/jQuery.dotdotdot#4f7d03a442ceca0b75a2fb641a065e47297d5666",
     "lodash": "^4.17.21",
     "lozad": "^1.16.0",
+    "npm": "^8.19.2",
```

Comment:
npm is added because of code build requirement?

Reply:
This is probably something that came along with Node 16, I honestly don't know why npm version *wasn't* tracked in Package.json before.